tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -153,7 +153,7 @@ describe('Home - Game List', () => {
 
 describe('Home - Create Game', () => {
   beforeEach(setup);
-  it('Creates a new game by hitting enter in text field', () => {
+  it.skip('Creates a new game by hitting enter in text field', () => {
```

Comment:
Will probably remove this test

Reply:
Opted to make this work and keep the test. Can submit new game form with enter key